src/firebase_setup/firebase.js

```diff
@@ -0,0 +1,30 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import {
+  createUserWithEmailAndPassword,
+  getAuth,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyBlnDkLusRKEg22hRPgbaIflea7688KfuQ",
+  authDomain: "login-auth-d513c.firebaseapp.com",
+  projectId: "login-auth-d513c",
+  storageBucket: "login-auth-d513c.appspot.com",
+  messagingSenderId: "599722188382",
+  appId: "1:599722188382:web:176496b15c64dc1b229ecf",
+  measurementId: "G-HT5F2WSMJS",
+};
```

Comment:
Please move this to your .env file and access it accordingly

```env
REACT_APP_FIREBASE_API_KEY=AIzaSyBlnDkLusRKEg22hRPgbaIflea7688KfuQ
REACT_APP_FIREBASE_AUTH_DOMAIN=login-auth-d513c.firebaseapp.com
REACT_APP_FIREBASE_PROJECT_ID=login-auth-d513c
REACT_APP_FIREBASE_STORAGE_BUCKET=login-auth-d513c.appspot.com
REACT_APP_FIREBASE_MESSAGING_SENDER_ID=599722188382
REACT_APP_FIREBASE_APP_ID="1:599722188382:web:176496b15c64dc1b229ecf"
```

In your firebase file 
```jsx
const firebaseConfig = {
  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
  // -- do for the rest for the keys
}```